src/CommunicationLayer/CommunicationLayer.pro

```diff
@@ -1,10 +1,14 @@
 TEMPLATE = lib
-CONFIG += staticlib
+CONFIG += staticlib conan_basic_setup
 
 ! include(../common.pri){
     error("Could not find common.pri file!")
 }
 
+! include($$OUT_PWD/../conanbuildinfo.pri) {
```

Comment:
Actually, `OUT_PWD` might not be what we want to use. Given that in theory, the build folder can be built anywhere, if I started my build in some completely different place outside of the src like `~/theShadowRealm/build`, then this wouldn't work.

I think you actually want to do `PWD`, which is the path to this file (`CommunicationLayer.pro`) instead.

Reply:
nvm ignore this, I got `conanbuildinfo.txt` and `conanfile.txt` confused